scripts/extract_jartic_2024_2025.py

```
@@ -0,0 +1,112 @@
+    print(f"Successfully processed: {len(successful)}/{len(all_archives)} archives")
+    
+    if successful:
+        print("\n✓ Successful:")
+        for archive in successful:
+            print(f"  - {archive}")
+    
+    if failed:
+        print("\n✗ Failed:")
+        for archive in failed:
+            print(f"  - {archive}")
+        print(f"\nTo retry failed archives, run them individually:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)